src/components/desktop/Header.js

```diff
@@ -51,9 +51,10 @@ const StyledHeader = styled.header`
     ul {
       list-style: none;
       padding: 0;
-      margin: 0 0 0 0;
+      margin: 0;
       height: 100%;
       display: flex;
+      align-items: center;
```

Comment:
This property makes the background look like this when Map is clicked:
![Screenshot from 2024-11-16 15-44-41](https://github.com/user-attachments/assets/68e4e94b-ef03-45b9-9e3e-d8d99f52aefc)